docker/non_root_run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+# last modified 10-05-2019
+set -ex
```

Comment:
This 'immediate fail and outut' can go higher up near the top of the `Dockerfile` as `RUN set -ex` - to help catch errors early on - i imagine the setting will then persist while running this script?!